lib/sailor.js

```diff
@@ -17,12 +18,29 @@ function Sailor(settings) {
     this.messagesCount = 0;
     this.amqpConnection = new amqp.AMQPConnection(settings);
     this.componentReader = new ComponentReader();
-    this.snapshot = this.getStepSnapshot();
-    this.apiClient = new ApiClient(settings.API_URI).auth(settings.API_USERNAME, settings.API_KEY);
+    this.snapshot = {};
+    this.stepData = {};
+    this.apiClient = RestApiClient(settings.API_USERNAME, settings.API_KEY);
```

Comment:
We need to set the API base url here in order to make sailor work on any environment. This way it will only work with api.elastic.io.

See https://github.com/elasticio/elasticio-rest-node/blob/master/lib/client.js#L3